### PR TITLE
add an ok computer check for the Links AMR client

### LINKED ISSUE #584: Implement an OK-computer check for WOS Links-AMR service
The Links-AMR service needs to be monitored along with the WOS-API endpoints.

### PR DESCRIPTION
**Pull request by peetucket**
fixes #584

### CODE REVIEW

**Comment by darrenleeweber** [comment]
There is no test for that method - that’s why the CC diff coverage is failing; see the specs/lib/web_of_science_spec for an example of testing the .working? method

**Comment by peetucket** [comment]
Trying to add a test for success/failure but having trouble with mocking the success response

File: spec/lib/clarivate/links_client_spec.rb [hunk 1]
```diff
@@ -98,4 +98,21 @@
...
   end
+
+  describe '.working?' do
+    context 'success' do
+      it 'returns true when it works (has 2 results)' do
+        response_xml = File.read('spec/fixtures/clarivate/links_response.xml')
+        allow(links_client.send(:connection)).to receive(:post).with(any_args).and_return(double(body: response_xml))
+        expect(described_class.working?).to be true
+      end
+    end
```
Comment (darrenleeweber) [inline spec/lib/clarivate/links_client_spec.rb]: Try 
```ruby
context 'success', :vcr do
  it 'returns true when it works (has 2 results)' do
    expect(described_class.working?).to be true
  end
end
```
The VCR cassette should have the results expected.  This spec might benefit from using VCR.

File: lib/clarivate/links_client.rb [hunk 2]
```diff
@@ -10,6 +10,13 @@ class LinksClient
 
     attr_reader :username, :password, :host
 
+    def self.working?
+      wos_uids = %w[WOS:A1976BW18000001 WOS:A1972N549400003]
+      links = new.links(wos_uids, fields: Clarivate::LinksClient::ALL_FIELDS)
+      raise 'Links AMR client did not return the correct number of records' unless links.is_a?(Hash) && links.keys.count == 2
+      true
+    end
```
Comment (darrenleeweber) [inline lib/clarivate/links_client.rb]: Do we care to get `Clarivate::LinksClient::ALL_FIELDS` for this?

**Comment by peetucket** [comment]
figured out the vcr issue, had to briefly uncomment mocking of auth code to record the cassette

**Pull request merged**